src/sx.ts

```diff
@@ -1,10 +1,12 @@
 import css, {SystemCssProperties, SystemStyleObject} from '@styled-system/css'
 import {ThemeColorPaths, ThemeShadowPaths} from './theme'
-import {ColorProps, ShadowProps} from 'styled-system'
+import {ColorProps, BorderColorProps, ShadowProps} from 'styled-system'
 import merge from 'deepmerge'
 
 export type BetterCssProperties = {
   [K in keyof SystemCssProperties]: K extends keyof ColorProps
+    ? ThemeColorPaths | SystemCssProperties[K]
+    : K extends keyof BorderColorProps
```

Comment:
@gr2m I know this is late, but I wonder if there's some tests we can use to ensure these don't regress, or to ensure they're fully expressive of options that work, and don't include options that don't work?

Reply:
Also I feel if we added tests, we would only test behavior of styled-system?

Reply:
there are some other type-tests in the repo.  I think since we're exposing a type that used in many places, it would be good to have some way to validate it, and ensure it's inclusive of all possible options.

I don't think tsd is used in the repo, but might be valuable to add something like it for type only assertions?

right now I think most type tests just ensure functions compile correctly - there's a bunch of files with `.types.test` suffixes where we do type only tests

Reply:
okay cheers I'll send a follow up pull request, but can't say when I get to it